jupyter_book/commands/__init__.py

```diff
@@ -9,7 +9,7 @@
 import subprocess
 from sphinx.util.osutil import cd
 
-from ..sphinx import build_sphinx
+from ..jbsphinx import build_sphinx
```

Comment:
why are you renaming this module?

Reply:
there's an issue with vscode running on windows -- the local sphinx module
shadows the global sphinx install so that importing sphinx.utils fails.
This was the path of least resistance around that problem, happy to revert if
necessary and just keep this on another fork.

Reply:
yeah I'd prefer not to rename the submodule if possible. You should be able to directly import anyway, `..sphinx` should be treated differently from `sphinx`

Reply:
yes, it's definitely a problem with the vscode debugger  -- there are a couple of vscode issues that I'll leave for someone who actually works with windows.